SU2_CFD/src/output/COutput.cpp

```diff
@@ -552,15 +550,34 @@ void COutput::WriteToFile(CConfig *config, CGeometry *geometry, unsigned short f
       surfaceDataSorter->SortConnectivity(config, geometry);
       surfaceDataSorter->SortOutputData();
 
-      /*--- Write paraview binary ---*/
+      /*--- Write surface tecplot binary ---*/
       if (rank == MASTER_NODE) {
-          (*fileWritingTable) << "Tecplot surface" << fileName + CTecplotBinaryFileWriter::fileExt;
+          (*fileWritingTable) << "Tecplot binary surface" << fileName + CTecplotBinaryFileWriter::fileExt;
       }
 
       fileWriter = new CTecplotBinaryFileWriter(volumeFieldNames, nDim, fileName, surfaceDataSorter,
                                                 curTimeIter, GetHistoryFieldValue("TIME_STEP"));
 
       break;
+      
+    case STL:
+
+     if (fileName.empty())
+       fileName = config->GetFilename(surfaceFilename, "", curTimeIter);
+
+       /*--- Load and sort the output data and connectivity. ---*/
```

Comment:
Looks like this is not aligned as intended.

Reply:
Yep, fixed that